openbas-front/src/admin/components/common/injects/Injects.tsx

```diff
@@ -505,112 +510,111 @@ const Injects: FunctionComponent<Props> = ({
+            classes={{ root: classes.itemHead }}
+            divider={false}
+            style={{ paddingTop: 0 }}
+          >
+            <ListItemIcon style={{ minWidth: 40 }}>
+              <Checkbox
+                edge="start"
+                checked={selectAll}
+                disableRipple
+                onChange={handleToggleSelectAll}
+                disabled={typeof handleToggleSelectAll !== 'function'}
               />
-            }
-          />
-          <ListItemSecondaryAction />
-        </ListItem>
-        {injects.map((inject: InjectOutputType, index) => {
-          const injectContract = inject.inject_injector_contract?.convertedContent;
-          const isContractExposed = injectContract?.config.expose;
-          return (
-            <ListItem
-              key={inject.inject_id}
-              classes={{ root: classes.item }}
-              divider
-              button
-              onClick={() => {
-                if (injectContract && isContractExposed) {
-                  setSelectedInjectId(inject.inject_id);
-                }
-              }}
-            >
-              <ListItemIcon
-                style={{ minWidth: 40 }}
-                onClick={(event) => (event.shiftKey
-                  ? onRowShiftClick(index, inject, event)
-                  : onToggleEntity(inject, event))
-                }
-              >
-                <Checkbox
-                  edge="start"
-                  checked={
-                    (selectAll && !(inject.inject_id
-                      in (deSelectedElements || {})))
-                    || inject.inject_id in (selectedElements || {})
-                  }
-                  disableRipple
+            </ListItemIcon>
+            <ListItemIcon />
+            <ListItemText
+              primary={
+                <SortHeadersComponentV2
+                  headers={headers}
+                  inlineStylesHeaders={inlineStyles}
+                  sortHelpers={queryableHelpers.sortHelpers}
                 />
-              </ListItemIcon>
-              <ListItemIcon style={{ paddingTop: 5 }}>
-                <InjectIcon
-                  isPayload={isNotEmptyField(inject.inject_injector_contract?.injector_contract_payload)}
-                  type={
-                    inject.inject_injector_contract?.injector_contract_payload
-                      ? inject.inject_injector_contract?.injector_contract_payload?.payload_collector_type
-                      || inject.inject_injector_contract?.injector_contract_payload?.payload_type
-                      : inject.inject_type
+              }
+            />
+            <ListItemSecondaryAction />
+          </ListItem>
+          {injects.map((inject: InjectOutputType, index) => {
+            const injectContract = inject.inject_injector_contract?.convertedContent;
+            const isContractExposed = injectContract?.config.expose;
+            return (
+              <ListItemButton
```

Comment:
You can't do that
Check this screenshot
![image](https://github.com/user-attachments/assets/0c4a84f2-54ec-4181-86a6-d20a71866da8)

VS
![image](https://github.com/user-attachments/assets/87761b9d-c68a-47af-958e-d34d9c025848)